src/objects/builtin_filter.rs

```diff
@@ -9,7 +9,7 @@ use crate::objects::syms::ObjectSyms;
 pub fn merge_required_objects(
     _obj_dir: &Path,
     merged_path: &Path,
-    objects: &HashMap<PathBuf, ObjectSyms>,
+    objects: &[PathBuf],
```

Comment:
Looks like those two imports (HashMap, ObjectSyms) just need to be remove to make Clippy CI pass

Reply:
Thanks, didn't think to run `clippy`. It's now passing locally.